test/TestInternalGrains/ReminderTestGrain2.cs

```diff
@@ -253,7 +278,15 @@ public override Task OnDeactivateAsync()
             if (validate)
                 r = await RegisterOrUpdateReminder(reminderName, /*TimeSpan.FromSeconds(3)*/usePeriod - TimeSpan.FromSeconds(2), usePeriod);
             else
-                r = await RuntimeClient.Current.RegisterOrUpdateReminder(reminderName, /*TimeSpan.FromSeconds(3)*/usePeriod - TimeSpan.FromSeconds(2), usePeriod);
+            {
+                var grainRef = this.AsWeaklyTypedReference();
+                var reminderService = this.runtimeClient.GetReminderService(
```

Comment:
The intention here is to emulate the original 'unvalidated' branch. I didn't check that there's actually any value to this behaviour.